Governance/tech/drs.md

```diff
@@ -1,4 +1,4 @@
-# Choosing a directory structure (DRS) and filenames
+# Choosing a directory structure: DRS and filenames
```

Comment:
I'd contemplate removing "DRS" from titles altogether in an effort to reduce jargon, what do you think?

Reply:
agreed